ql/src/java/org/apache/hadoop/hive/ql/exec/repl/util/ReplUtils.java

```diff
@@ -60,6 +60,7 @@
 
   public static final String LAST_REPL_ID_KEY = "hive.repl.last.repl.id";
   public static final String REPL_CHECKPOINT_KEY = "hive.repl.ckpt.key";
+  public static final String REPL_FIRST_INC_PENDING_FLAG = "hive.repl.first.inc.pending";
```

Comment:
Can we use generic name like "hive.repl.is.data.consistent"? This would avoid confusion for incremental bootstrap case of table level replication where few tables data will be consistent only after next incremental load.

Reply:
add a comment to explain

Reply:
add a TODO for external bootstrap during inc load

Reply:
OK